compiler/main/runpasses.cpp

```diff
@@ -165,6 +166,11 @@ void runPasses(PhaseTracker& tracker, bool isChpldoc) {
   size_t passListSize = sizeof(sPassList) / sizeof(sPassList[0]);
 
   setupLogfiles();
+  //vass - here because I am patching this file already
+  AstDumpToNode::compact = true;
+  AstDumpToNode::delimitEnter = "(";
+  AstDumpToNode::delimitExit  = ")";
```

Comment:
Huh? Maybe you should just delete this change? (Or, at least, I don't think it belongs in this PR)